extensions/sql-database-projects/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "sql-database-projects",
   "displayName": "SQL Database Projects",
   "description": "Enables users to develop and publish database schemas for MSSQL Databases",
-  "version": "1.3.1",
```

Comment:
Why was this 1.3.1? We didn't provide a hotfix in the last release, right?

Reply:
It's because 1.3.0 was released in insiders a couple weeks before the September release so that partners could test one of the new target platforms, then there were a couple more changes that were made after that so the version was bumped to 1.3.1 for the September release.

Reply:
Oh makes sense. Thank you for the explanation :).